src/router/index.js

```diff
@@ -23,4 +63,16 @@ const router = new VueRouter({
   routes,
 });
 
+router.beforeEach((to, from, next) => {
+  if (to.matched.some((record) => record.meta.requiresAuth)) {
+    if (store.getters.isLoggedIn) {
+      next();
+      return;
+    }
+    next('/login');
+  } else {
+    next();
+  }
```

Comment:
Kan wel iets netter.
`if (to.matched.some((record) => record.meta.requiresAuth) && !store.getters.isLoggedIn) {
  next('login');
} else {
  next();
}`

Reply:
Done